src/Widgets/NotificationEntry.vala

```diff
@@ -78,17 +80,36 @@ public class Notifications.NotificationEntry : Gtk.ListBoxRow {
             column_spacing = 6,
             margin = 12,
             // Box shadow is clipped to the margin area
-            margin_top = 1,
+            margin_top = 14,
             margin_bottom = 11
         };
 
         unowned Gtk.StyleContext grid_context = grid.get_style_context ();
         grid_context.add_class (Granite.STYLE_CLASS_CARD);
         grid_context.add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
 
+        var delete_image = new Gtk.Image.from_icon_name ("window-close-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
+        delete_image.get_style_context ().add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+
+        var delete_box = new Gtk.Button () {
```

Comment:
Can we call this `delete_button` instead since it's a Gtk.Button subclass and not a Gtk.Box subclass